Assets/Scripts/UI/WarningText.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class WarningText : MonoBehaviour
+{
+    private enum alphaValue
+    {
+        SHRINKING,
+        GROWING,
+    }
+
+    private alphaValue currentAlphaValue;
+    [SerializeField] private float CommentminAlpha;
+    [SerializeField] private float CommentmaxAlpha;
+    [SerializeField] private float CommentCurrentAlpha;
+    [SerializeField] private bool flashText;
+
+    [SerializeField] private Text MyText;
+
+    void Start()
+    {
+        CommentminAlpha = 0f;
+        CommentmaxAlpha = 1.0f;
+        CommentCurrentAlpha = 1.0f;
+        currentAlphaValue = alphaValue.SHRINKING;
+    }
+
+    void Update()
+    {
+        if (flashText == true)
+        {
+            alphaComments();
+        }
+        
+    }
+
+    public void SetIsAllowedToFlash(bool textFlash)
+    {
+        flashText = textFlash;
+    }
+
+    private void alphaComments()
+    {
+        if (currentAlphaValue == alphaValue.SHRINKING)
+        {
+            CommentCurrentAlpha = CommentCurrentAlpha - 0.005f;
+            MyText.color = new Color(Color.white.r, Color.white.g, Color.white.b, CommentCurrentAlpha);
+            if (CommentCurrentAlpha <= CommentminAlpha)
+            {
+                currentAlphaValue = alphaValue.GROWING;
+            }
+        }
+        else if (currentAlphaValue == alphaValue.GROWING)
+        {
+            CommentCurrentAlpha = CommentCurrentAlpha + 0.005f;
+            MyText.color = new Color(Color.white.r, Color.white.g, Color.white.b, CommentCurrentAlpha);
+            if (CommentCurrentAlpha >= CommentmaxAlpha)
+            {
+                currentAlphaValue = alphaValue.SHRINKING;
+            }
+        }
+    }
```

Comment:
```suggestion
    private void alphaComments()
    {
        if (currentAlphaValue == alphaValue.SHRINKING)
        {
            CommentCurrentAlpha = CommentCurrentAlpha - flashingSpeed;
            text.color = new Color(Color.white.r, Color.white.g, Color.white.b, CommentCurrentAlpha);
            if (CommentCurrentAlpha <= CommentminAlpha)
            {
                currentAlphaValue = alphaValue.GROWING;
            }
        }
        else if (currentAlphaValue == alphaValue.GROWING)
        {
            CommentCurrentAlpha = CommentCurrentAlpha + flashingSpeed;
            text.color = new Color(Color.white.r, Color.white.g, Color.white.b, CommentCurrentAlpha);
            if (CommentCurrentAlpha >= CommentmaxAlpha)
            {
                currentAlphaValue = alphaValue.SHRINKING;
            }
        }
    }
```

Je heb nu een magic number 0.005, je kan hiervan een serzildezfield float van maken en als je het perfect wilt hebben kan je het ook een range geven, want 0.5 was te snel.